### DISCUSSION TITLE
集群管理能力在1.1版本和1.3版本的差异

### DISCUSSION THREAD

**Discussion by steven6718** [kubevela/kubevela#4084] [Q&A] [answered]
你好，之前测试vela1.1版本的多集群管理能力，在集群join后，会在子集群安装一个crd：resourcetrackers.core.oam.dev和一个APIService：v1beta1.core.oam.dev，但是在测试1.3版本的多集群管理时，发现join后不会安装这两个资源，咨询下这个是正常的吗？是升级后resourcetracker都在控制面集群管理了么？还是我的join操作有问题，导致对应的资源没有自动创建？辛苦帮忙解惑~

**Comment by wonderflow** [comment, marked as answer]
正常的，1.3 去掉了这种依赖，子集群更干净。
